src/pages/Lend/Filter/FilterRangeSlider.vue

```diff
@@ -1,66 +1,60 @@
 <template>
-	<ais-range-input :attribute="queryAttribute" :min="minimumValue" :max="maximumValue">
-		<div slot-scope="{ range, refine }">
-			<div class="header">
-				<span class="title">
-					{{ title }}
-				</span>
-				<span class="range-label">
-					({{ rangeLabel }})
-				</span>
-			</div>
+	<div>
+		<div class="header">
+			<span class="title">
+				{{ title }}
+			</span>
+			<span class="range-label">
+				({{ rangeLabel }})
+			</span>
+		</div>
+		<div class="slider">
 			<input
-				:id="id"
 				type="range"
 				:min="minimum"
 				:max="maximum"
-				:value="rangeValue"
+				:value="sliderValue"
 				:style="sliderStyle"
 				@input="onSliderChange(
 					refine,
 					range,
-					$event)"
-			>
+					$event)">
+
+			<div class="hide">
+				{{ sliderPosition }}
```

Comment:
Lets add a comment with a `// TODO: ` signifying that this was added to make the component reactive as well as a quick reminder of why we needed to add a hidden div and why line 132 doesn't need to return anything